core/deployment/src/main/java/io/quarkus/artemis/core/deployment/DevServicesArtemisProcessor.java

```diff
@@ -72,6 +73,17 @@ public List<DevServicesResultBuildItem> startArtemisDevService(
             DevServicesConfig devServicesConfig) {
         ArrayList<DevServicesResultBuildItem> results = new ArrayList<>();
         for (String name : bootstrap.getConfigurationNames()) {
+            String propertyName = name;
+            for (String rawName : ConfigProvider.getConfig().getPropertyNames()) {
+                if (rawName.contains(name)) {
+                    String str = rawName.substring("quarkus.artemis.".length());
+                    str = str.substring(0, str.indexOf('.'));
+                    if (str.replace("\"", "").replace("'", "").equals(name)) {
```

Comment:
Should we use `String.replace(char, char)` here?

Reply:
No, we need to remove the quotes here.

Reply:
I think we don't need this anymore